packages/presentation/src/components/MessageViewer.svelte

```diff
@@ -28,6 +31,12 @@
   }
 </script>
 
-<div class="text-markup-view">
-  <Node {node} {preview} />
-</div>
+{#if !isMarkdown}
+  <div class="text-markup-view">
+    <Node {node} {preview} />
+  </div>
+{:else}
+  <div class="markdown-view">
+    <Html value={marked(message, { async: false })} />
+  </div>
+{/if}
```

Comment:
It looks like `node` and `empty` are not going to be used for the `isMarkdown` branch, moreover, the `MessageViewer.svelte` looks  simple. 

Just curious, is it possible to use this code _inline_ in`MailThread.svelte` and do not modify  `ChatMessagePresenter.svelte`?

Reply:
In this case all other parts of `ChatMessagePresenter` will not be reused: author avatar, date, styles, etc.